runtime/etc/Makefile.comm-ofi

```diff
@@ -16,7 +16,14 @@
 # limitations under the License.
 
 ifneq ($(LIBFABRIC_DIR),)
-GEN_LFLAGS += -L$(LIBFABRIC_DIR)/lib -Wl,-rpath,$(LIBFABRIC_DIR)/lib
+  LIBFAB_DIR_CACHE := $(shell if [ -d $(LIBFABRIC_DIR)/lib64 ] ; then \
```

Comment:
I have a slight preference for `LIBFAB_LIB_DIR` or something instead of CACHE

Reply:
Could use wildcard instead of shelling out. We do this in other scripts, something like:

```Make
ifneq ("$(wildcard $(LIBFABRIC_DIR)/lib64)","")
  LIBFAB_LIB_DIR := $(LIBFABRIC_DIR)/lib64
else ifneq ("$(wildcard $(LIBFABRIC_DIR)/lib)","")
  LIBFAB_LIB_DIR := $(LIBFABRIC_DIR)/lib
endif
```